tests/integration-tests/tests/dns/test_dns.py

```diff
@@ -98,7 +98,7 @@ def test_existing_hosted_zone(
 
     # Test domain name matches expected domain name
     resolv_conf = remote_command_executor.run_remote_command("cat /etc/resolv.conf").stdout
-    assert_that(resolv_conf).contains(cluster.cfn_name + "." + domain_name)
```

Comment:
let's extend an integration tests to create a cluster with an UPPERCASE cluster name, in this way we're sure all these changes are working as expected.

Reply:
we could also modify the current test to create a cluster with an uppercase letter. please leave the integ-tests prefix and the suffix unaltered

Reply:
Good idea! Modify cluster_factoryAdd 2 integration test test_slurm and test_existing_hosted_zone to include uppercase in clustername

Reply:
awesome thanks